us/en/skymiles/styles/styles.css

```diff
@@ -481,6 +482,10 @@ main img {
   main .section {
     max-width: 375px;
   }
+
+  main .section.accordion-container {
+    width: 100%;
```

Comment:
Not sure this would have any effect since sections should already be displayed as block by default, and hence have 100% width

Reply:
fixed in [f1d8070](https://github.com/hlxsites/delta/pull/89/commits/f1d80700d4d0f621c6466bffc47d9c5b6e4f461c)